docs/Overview/SupportedDataSources.md

```diff
@@ -11,16 +11,16 @@ Apache DevLake(incubating) supports the following data sources. The data from ea
-| BitBucket                       | Source Code Management, Code Review                        | Cloud                                                         | Not Available          | `bitbucket`, `gitextractor` | Full Refresh                                                   |
+| GitHub | Source Code Management, Code Review, Issue Tracking, CI/CD (GitHub Actions) | Cloud                                                         | Available              | `github`, `gitextractor`    | Full Refresh                                                   |
+| GitLab | Source Code Management, Code Review, Issue Tracking, CI/CD (GitLab CI) | Cloud, Community Edition 13.x+                                | Available              | `gitlab`, `gitextractor`    | Full Refresh, Incremental Sync(for `issues`,`MRs`)             |
 | Jira                            | Issue Tracking                                             | Cloud, Server 7.x+, Data Center 7.x+                          | Available              | `jira`                      | Full Refresh, Incremental Sync(for `issues` and related)       |
-| TAPD                            | Issue Tracking                                             | Cloud                                                         | Not Available          | `tapd`                      | Full Refresh, Incremental Sync(for `stories`, `bugs`, `tasks`) |
 | Jenkins                         | CI/CD                                                      | 2.263.x+                                                      | Available              | `jenkins`                   | Full Refresh                                                   |
+| Gitee                           | Source Code Management, Code Review, Issue Tracking        | Cloud                                                         | Not Available          | `gitee`, `gitextractor`     | Full Refresh, Incremental Sync(for `issues`,`MRs`)             |
+| BitBucket                       | Source Code Management, Code Review                        | Cloud                                                         | WIP          | `bitbucket`, `gitextractor` | Full Refresh                                                   |
+| TAPD                            | Issue Tracking                                             | Cloud                                                         | Not Available          | `tapd`                      | Full Refresh, Incremental Sync(for `stories`, `bugs`, `tasks`) |
 | Feishu                          | Calendar                                                   | Cloud                                                         | Not Available          | `feishu`                    | Full Refresh                                                   |
```

Comment:
| ZENTAO                            | Issue Tracking                                             | Cloud                                                         | Not Available          | `zentao`                      | Full Refresh |